src/resolve/resolvectl.c

```diff
@@ -1156,8 +1156,9 @@ static int reset_server_features(int argc, char **argv, void *userdata) {
 }
 
 typedef enum ReadDnsServerFlag {
```

Comment:
Comment about the commit itself: it'd help a lot if the result of the change was copy-pasted into the commit message.